src/modules/staff-checker.js

```diff
@@ -43,7 +44,7 @@ module.exports = {
                     httpsRequest('script.google.com', process.env.APPS_SCRIPT_PATH, data)
                 }
                 lastStaffSeen = Date.now();
-                lastStaffMessage.edit(`Last staff member seen at:\n${new Date()}`);
+                lastStaffMessage.edit(`Last staff member seen at:\n<t:${new Date().getTime()}:F> (<t:${new Date().getTime()}:R>)`);
```

Comment:
This is parsed on startup so make sure it's still able to be read

Reply:
uhhh what do you mean by that though?

Reply:
The bot reads this message and stores it as the last time a staffer seen, so that it doesn't break across restarts

Reply:
ohhh

that should be fixed now